python/python_fastapi_server/routers/covjsonresponse.py

```diff
@@ -0,0 +1,13 @@
+import json
+from typing import Any
+
+from fastapi import Response
+from fastapi.encoders import jsonable_encoder
+
+
+class CovJSONResponse(Response):
+    media_type = "application/prs.coverage+json"
+
+    def render(self, content: Any) -> bytes:
+        return content.json(exclude_none=True)
+
```

Comment:
This is no longer needed with pydantic 2.0. Use this:
```suggestion
class CoverageJsonResponse(JSONResponse):
    media_type = ResponseFormatOptions.coveragejson.value

```